server/game/cards/13.6-LMHR/PoisonedDagger.js

```diff
@@ -0,0 +1,38 @@
+const DrawCard = require('../../drawcard.js');
+
+class PoisonedDagger extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ controller: 'current' });
+        this.whileAttached({
+            effect: ability.effects.modifyStrength(2)
+        });
+        this.reaction({
+            when: {
+                afterChallenge: event => event.challenge.winner === this.controller && this.parent.isParticipating() && event.challenge.challengeType === 'intrigue'
+            },
+            cost: ability.costs.sacrificeSelf(), 
+            handler: context => {
+                let otherPlayer = context.event.challenge.loser;
+                this.game.promptForSelect(otherPlayer, {
+                    cardCondition: card => (
+                        card.isParticipating() &&
+                        card.getType() === 'character' &&
+                        card.controller === otherPlayer),
+                    activePromptTitle: 'Select character to kill',
+                    source: this,
+                    gameAction: 'kill',
+                    onSelect: (player, card) => {

```

Comment:
There should also be an `onCancel` here that adds an alert in the game log saying the player cancelled the resolution of the ability.

Reply:
i looked at ser mandon moore for the onCancel handler as i figured it would be worth an alert if the player does not choose any characters.